providers/chain-config.ts

```diff
@@ -1,19 +1,20 @@
 import { createConfig, fallback, http } from 'wagmi';
 import { getDefaultConfig } from 'connectkit';
-import { gnosis } from 'wagmi/chains';
+import { gnosis, mainnet } from 'wagmi/chains';
 import { safe } from 'wagmi/connectors';
 import { ChainId, RPC_LIST } from '@/constants';
 
 const defaultConfig = getDefaultConfig({
-  chains: [gnosis],
+  chains: [gnosis, mainnet],
```

Comment:
Do we need this? I'm not sure if we need it. This makes it possible to connect to ethereum and not show some kind of error. Example screenshot:

![image](https://github.com/user-attachments/assets/7c7e8593-5eed-4cac-b163-b72db5245ffb)

Reply:
the only way I found to make the ens work, cause they have to pull data from `mainnet` and not having it on config would only make it work on your own ens.

Reply:
Took a quick look, let's remove mainnet from this default config.

And let's create a new config, called `mainnetConfig` and use it with `useEnsName` only.

Here is the example:

```
export const mainnetConfig = createConfig({
  chains: [mainnet],
  transports: {
    [mainnet.id]: fallback([http(RPC_LIST[ChainId.MAINNET]), http()]),
  },
});
```

Then let's pass it on `useEnsName`

```
const { data: ensName } = useEnsName({
    address: normalizedAddress,
    chainId: mainnet.id,
    config: mainnetConfig,
  });
  ```